xlators/features/bit-rot/src/stub/bit-rot-stub.c

```diff
@@ -2859,6 +2859,9 @@ br_stub_readdirp(call_frame_t *frame, xlator_t *this, fd_t *fd, size_t size,
     if (frame->local == (void *)0x1)
         frame->local = NULL;
     STACK_UNWIND_STRICT(readdirp, frame, -1, op_errno, NULL, NULL);
+
+    if (xref)
+        dict_unref(dict);
```

Comment:
This is not necessary. Just remove the next `return` to go to `unref_dict` which will unref the dict if necessary and return.

Reply:
done